markdown/developers/adding_pipelines.md

```diff
@@ -168,13 +168,22 @@ in the `.github/workflows/` YAML files too.
 Ok, so you're essentially finished. Your pipeline is written, the tests pass and
 you're ready to add your workflow to nf-core.
 
+To add your pipeline you have two options, transferring ownership or forking.
+
+### Forking
```

Comment:
We should just remove this. I don't think there's ever a time when we want to fork repos.